setup.py

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/python3
 # -*- coding: utf-8 -*-
-#--------------------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------------------------------
```

Comment:
Just curiosity: is it somehow problematic to have "#-" in the code?

Reply:
No, sorry, that was from my linter. I’ll update.